code/models.py

```diff
@@ -0,0 +1,52 @@
+'''
```

Comment:
I think we should just steal the classes in `model.py` from the Stanford code and add a `generate` method to `LanguageModel`